### DISCUSSION TITLE
Installing and opening Jupyter Lab

### DISCUSSION THREAD

**Discussion by Farid-git7** [giswqs/geog-312#67] [Q&A] [answered]
I have tried to install geemap and leafmap through both conda and pip ; conda fails to to install but pip seems to have finished the process. 
all in all, I can not open the Jupyter lab as the error shown in the attached Screenshot. Help please
![Screenshot 2024-09-19 231456](https://github.com/user-attachments/assets/3c2a24e8-b39e-4d84-bc3f-707762dbbdbc)

**Comment by giswqs** [comment, marked as answer]
```bash
pip install jupyterlab
```
